src/parser/csql_grammar.y

```diff
@@ -142,11 +142,19 @@ extern int yybuffer_pos;
 #define PRINT_(a) printf(a)
 #define PRINT_1(a, b) printf(a, b)
 #define PRINT_2(a, b, c) printf(a, b, c)
+#define DBG_PRINT_MATCH(...)    do { fprintf(stdout, "  *** RULE) "__VA_ARGS__);  fflush(stdout); } while(0)
+#define DBG_PRINT_MATCH_LN(...) do {                        \
+                fprintf(stdout, "  *** RULE) "__VA_ARGS__); \
+                fprintf(stdout, "\n");                      \
+                fflush(stdout);                             \
+              } while(0)
```

Comment:
The added macro is a new addition because it is difficult to know which part is being processed with the existing DBG_PRINT.
Since then, there is a need to keep adding, so we want to keep it going.